src/historywork/GetAndUnzipRemoteFileWork.h

```diff
@@ -13,7 +14,7 @@ namespace stellar
 class HistoryArchive;
 class GetRemoteFileWork;
 
-class GetAndUnzipRemoteFileWork : public Work
```

Comment:
I think `GetHistoryArchiveStateWork` should also inherit from `DownloadWorkWithMetrics`.